pypesto/ensemble/ensemble.py

```diff
@@ -160,6 +163,11 @@ def compute_summary(self,
         if not self.prediction_results:
             raise ArithmeticError('Cannot compute summary statistics from '
                                   'empty prediction results.')
+        # check if weightings shall be used or not depending on whether llh
+        # is part of the prediciton output
+        if AMICI_LLH not in self.prediction_results[0].conditions[0].output:
+            weighting = False
```

Comment:
A user might explicitly want weights but not realize they're not used because of this. A warning might help?